### PR TITLE
MTDSA-6591 Request and Response models for amend employment financial…

### PR DESCRIPTION
**Pull request by Taiwo-AB**
… details

### CODE REVIEW

File: test/v1/models/request/amendFinancialDetails/employment/AmendBenefitsInKindSpec.scala [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    qualifyingRelocationExpenses = Some(78.24),
+    employerProvidedProfessionalSubscriptions = Some(84.56),
+    employerProvidedServices = Some(56.34),
+    incomeTaxPaidByDirector = Some(67.34),
+    travelAndSubsistence = Some(56.89),
+    vouchersAndCreditCards = Some(34.90),
+    nonCash = Some(23.89)
+  )
+
+  "AmendBenefitsInKind" when {
```
Comment (david-walters-HMRC) [inline test/v1/models/request/amendFinancialDetails/employment/AmendBenefitsInKindSpec.scala:91]: need another test for the second format

File: app/v1/models/request/amendFinancialDetails/emploment/AmendBenefitsInKind.scala [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                               vouchersAndCreditCards: Option[BigDecimal],
+                               nonCash: Option[BigDecimal])
+
+object AmendBenefitsInKind {
+  val empty: AmendBenefitsInKind = AmendBenefitsInKind(
+    None, None, None, None, None, None, None, None, None, None, None, None, None, None,
+    None, None, None, None, None, None, None, None, None, None, None, None, None, None
+  )
+
+  val firstFormat: OFormat[(
```
Comment (david-walters-HMRC) [inline app/v1/models/request/amendFinancialDetails/emploment/AmendBenefitsInKind.scala:57]: is there not better names rather then first format and second format?

File: test/v1/models/request/amendFinancialDetails/employment/AmendBenefitsInKindSpec.scala (hunk 1 above)
Comment (Taiwo-AB) [reply test/v1/models/request/amendFinancialDetails/employment/AmendBenefitsInKindSpec.scala:91]: Don't need another test. We try not write multiple tests that are testing the same behaviour.

File: app/v1/models/request/amendFinancialDetails/emploment/AmendBenefitsInKind.scala (hunk 2 above)
Comment (Taiwo-AB) [reply app/v1/models/request/amendFinancialDetails/emploment/AmendBenefitsInKind.scala:57]: What's wrong with first and second format?

**Pull request merged**